src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -22,25 +24,183 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 
 public class MythicMobsBridge extends Bridge {
 
+    static class MythicMobsBridgeTags extends PseudoObjectTagBase<MythicMobsBridgeTags> {
+
+        public static MythicMobsBridgeTags instance;
+
+        public MythicMobsBridgeTags() {
+            instance = this;
+            TagManager.registerStaticTagBaseHandler(MythicMobsBridgeTags.class, "mythicmobs", (t) -> instance);
+        }
+
+        @Override
+        public void register() {
+
+            // <--[tag]
+            // @attribute <mythicmobs.item_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicItem IDs. See also <@link tag mythic_item>.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "item_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getItemManager().getItemNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.skills>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicSkill IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "skills", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getSkillNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicMob IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "mob_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getMobManager().getMobNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.active_mobs>
+            // @returns ListTag(MythicMobsMobTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all active MythicMobs on the server.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "active_mobs", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (ActiveMob entity : getMobManager().getActiveMobs()) {
+                    list.addObject(new MythicMobsMobTag(entity));
+                }
+                return list;
```

Comment:
Should use `ListTag`'s new convertor constructor, and also update all other relevant places in the PR